themes/base/core.css

```diff
@@ -68,15 +68,18 @@
 
 /* Icons
 ----------------------------------*/
-
-/* states and images */
 .ui-icon {
-	display: block;
-	text-indent: -99999px;
-	overflow: hidden;
-	background-repeat: no-repeat;
+	display: inline-block;
+	vertical-align: middle;
+	margin-top: -.25em;
+	position: relative;
 }
 
+.ui-icon-display-block {
```

Comment:
This has more than just `display: block`. Let's find a name that captures where this class should be used, not what it does.